### PR TITLE
python3Packages.pycfdns: init at 1.2.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module for updating Cloudflare DNS A records

https://github.com/ludeeus/pycfdns

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pycfdns/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+, async-timeout
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "pycfdns";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
```
Comment (dotlambda) [inline pkgs/development/python-modules/pycfdns/default.nix:12]: Why use `fetchFromGitHub` if there are no tests?

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110592` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.pycfdns</li>
<li>python38Packages.pycfdns</li>
<li>python39Packages.pycfdns</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110592` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.pycfdns</li>
<li>python38Packages.pycfdns</li>
<li>python39Packages.pycfdns</li>
</ul>
</details>

File: pkgs/development/python-modules/pycfdns/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pycfdns/default.nix:12]: I prefer to use the upstream location of the source when ever possible.

**Pull request merged**